data/sql/updates/pending_db_auth/rev_1546540718864817294.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO version_db_auth (`sql_rev`) VALUES ('1546540718864817294');
+
+ALTER TABLE `account` CHANGE `last_login` `last_login` TIMESTAMP NULL;
```

Comment:
this should be like this
```alter table account change last_login last_login timestamp null default null```
it's a more clean way of doing it.

Reply:
lol you make colum null by default i'm not. no clean or dirty

Reply:
Doing it the way you have it in this PR, will just repeat the issue you're trying to close. So your PR is just useless without a proper method of fixing it.

Reply:
Yes, default should be Null I guess

Reply:
`last_login` timestamp NULL DEFAULT NULL,`